db/migrate/20240815215102_change_convert2_rhel_to_boolean.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeConvert2RhelToBoolean < ActiveRecord::Migration[6.1]
+  def up
+    change_column :katello_subscription_facets, :convert2rhel_through_foreman, :boolean, using: 'convert2rhel_through_foreman::boolean'
```

Comment:
`convert2rhel_through_foreman` is a tenary (according to the comment above). It can be `nil` if the fact is not present, and 0/1 if the fact is.
Wouldn't converting it to `bool` loose that detail? Or can the column still be `NULL`?

Reply:
https://api.rubyonrails.org/v7.2.0/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_column implies it's not specified in the resulting SQL so it's up to the DB implementation. Reading https://www.postgresql.org/docs/current/sql-altertable.html I get the impression the default is to allow `NULL`, which also makes sense: adding a column means you either need to provide a default value or  allow `NULL` values.